lib/fizzy/instructions.cpp

```diff
@@ -444,6 +444,222 @@ constexpr uint8_t instruction_max_align_table[256] = {
     /* f32_reinterpret_i32 = 0xbe */ 0,
     /* f64_reinterpret_i64 = 0xbf */ 0,
 };
+
+constexpr int16_t instruction_cost_table[256] = {
+    // 5.4.1 Control instructions
+    /* unreachable         = 0x00 */ 1,
```

Comment:
You can use identifiers for initialization, like in https://github.com/ethereum/evmone/blob/master/lib/evmone/instruction_traits.hpp#L42.

Reply:
Would need to change other tables in this file, too, then. Will do that in another PR first.

Reply:
Don't do it then.